schema_transformations.py

```diff
@@ -722,6 +768,36 @@ def correctify_orcid(org_orcid):
     return "https://orcid.org/{}".format(orcs[-1])
 
 
+def correctify_scopus(org_scopus):
+    """Correct illformatted Scopus."""
+    # Get rid of all spaces.
+    new_scopus = org_scopus.replace(' ', '')
+
+    if not re.search("^\d{1,11}$", new_scopus):
+        # Return original value.
+        return org_scopus
```

Comment:
Don't we need to return None in this case, so that calling functions know that the value can't be transformed, and they can log a warning message? Can we also capture the behaviour of this function in a unit test?

Reply:
Moved correctify functions to schema_transformation_utils and and added unit tests for them